prediction_market_agent/development_tools/prophet_agent_tester.py

```
@@ -100,23 +142,90 @@ def test_prophet_agent(
+                    )
+                    omen_agent_market_before_placing_bet = (
+                        OmenAgentMarket.from_data_model(market_before_placing_bet)
+                    )
+
+                    buy_trade_in_tokes = (
+                        omen_agent_market_before_placing_bet.get_in_token(
+                            trades[0].amount
+                        )
+                    )
+                    probs = check_not_none(prediction.outcome_prediction).probabilities
+                    predicted = max(probs, key=lambda k: probs[k])
```

Ruff (0.11.9)

Local variable `predicted` is assigned to but never used

Remove assignment to unused variable `predicted`

(F841)